### PR TITLE
Fixed SAIC-373: image debug logs

### PR DESCRIPTION
**Pull request by realsystem**
Added new debug messages for image migration.
Added progress bar.

### CODE REVIEW

File: cloudferrylib/utils/file_like_proxy.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 
+from progressbar import Bar, Percentage, ProgressBar
```
Comment (miarmak) [inline cloudferrylib/utils/file_like_proxy.py]: It is recommended  to not importing objects, only modules.
http://docs.openstack.org/developer/hacking/#imports

File: requirements.txt [hunk 2]
```diff
@@ -19,5 +19,6 @@ requests==2.2.1
 sqlalchemy
 oslo.config==1.15.0
 oslo.utils==1.9.0
+progressbar==2.3
```
Comment (miarmak) [inline requirements.txt]: Arrange it in alphabetical order please.

**Comment by miarmak** [comment]
Looks good, except of 2 little style issues inline.

**Comment by roman-verchikov** [comment]
+1

File: cloudferrylib/os/image/glance_image.py [hunk 3]
```diff
@@ -409,6 +412,7 @@ def deploy(self, info, callback=None):
         if migrate_images_list:
             im_name_list = [(im.name, tmp_meta) for (im, tmp_meta) in
                             migrate_images_list]
+            LOG.debug("images on destination")
```
Comment (roman-verchikov) [inline cloudferrylib/os/image/glance_image.py]: I would expect this message to have a list of images migrated, please update

**Comment by Settis** [comment]
Looks good. +1

**Comment by roman-verchikov** [comment]
+1

**Pull request merged**